app/controllers/OnlineTestController.scala

```diff
@@ -65,7 +65,7 @@ trait OnlineTestController extends BaseController {
   def getOnlineTest(userId: String) = Action.async { implicit request =>
```

Comment:
I think this method is specific to phase1, should we renamed to something like getPhase1OnlineTest of getPhase1Test?